<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaritasPDRARiskDialog</class>
 <widget class="QDialog" name="CaritasPDRARiskDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Compute Hazard, Vulnerability, Capacity, and Risk from PDRA Data (gpkg-version)</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/plugins/caritas_pdra/img/icons/icon-risk.png</normaloff>:/plugins/caritas_pdra/img/icons/icon-risk.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QgsMapLayerComboBox" name="selectHHComboBox">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>10</y>
     <width>441</width>
     <height>27</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Piliin ang household layer na naglalaman ng PDRA indicators</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="whatsThis">
    <string>Select the household point layer with PDRA indicators</string>
   </property>
   <property name="showCrs">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="selectHHLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Select Household Layer with PDRA Indicators</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="resultBasenameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>201</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Basename for Resulting Layers</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="resultBasenameLineEdit">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>50</y>
     <width>441</width>
     <height>27</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Ang mga resulta ay ilalagay sa memory at papangalanang &lt;basename_HAZ&gt;, &lt;basename_VUL&gt;, &lt;basename_CAP&gt;, and &lt;basename_RISK&gt;.</string>
   </property>
   <property name="whatsThis">
    <string>Resulting layers (in memory) will be named &lt;basename_HAZ&gt;, &lt;basename_VUL&gt;, &lt;basename_CAP&gt;, and &lt;basename_RISK&gt;.</string>
   </property>
  </widget>
  <widget class="QLabel" name="selectIndicatorsLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Select Indicators/Fields to Compute Risk From</string>
   </property>
  </widget>
  <widget class="QLabel" name="hazLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Hazard</string>
   </property>
  </widget>
  <widget class="QListWidget" name="hazListWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Piliin ang mga indikador ng Hazard (Posibilidad ng Sakuna) na isasama sa pagkuha ng Risk (Panganib).</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QListWidget" name="vulListWidget">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>150</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Piliin ang mga indikador ng Vulnerability (Kahinaan) na isasama sa pagkuha ng Risk (Panganib).</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="capListWidget">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>150</y>
     <width>281</width>
     <height>291</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Piliin ang mga indikador ng Capacity (Kapasidad) na isasama sa pagkuha ng Risk (Panganib).</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="selectionMode">
    <enum>QAbstractItemView::ExtendedSelection</enum>
   </property>
  </widget>
  <widget class="QLabel" name="vulLabel">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Vulnerability</string>
   </property>
  </widget>
  <widget class="QLabel" name="capLabel">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>120</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Capacity</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hazSelectAllBtn">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Piliin lahat</string>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="hazUnselectAllBtn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Alisin ang mga napili</string>
   </property>
   <property name="text">
    <string>Unselect All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vulUnselectAllBtn">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Alisin ang mga napili</string>
   </property>
   <property name="text">
    <string>Unselect All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="vulSelectAllBtn">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Piliin lahat</string>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capUnselectAllBtn">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Alisin ang mga napili</string>
   </property>
   <property name="text">
    <string>Unselect All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="capSelectAllBtn">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>450</y>
     <width>101</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Piliin lahat</string>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>5</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:6pt;&quot;&gt;This plugin was made possible due to the efforts of NASSA/Caritas Philippines.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>530</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/plugins/caritas_pdra/img/logos/logo-nassa-caritas.png</pixmap>
   </property>
  </widget>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Perform Risk Computation</string>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsMapLayerComboBox</class>
   <extends>QComboBox</extends>
   <header>qgsmaplayercombobox.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
